runtime.md

```diff
@@ -9,8 +9,11 @@ $ funC [global-options] <COMMAND> [command-specific-options] <command-specific-a
 
 ## Global options
 
-None are required, but the runtime may support options that start with at least one hyphen.
-Global options may take positional arguments (e.g. `--log-level debug`), but the option parsing must be such that `funC <COMMAND>` is unambiguously an invocation of `<COMMAND>` for any `<COMMAND>` that does not start with a hyphen (including commands not specified in this document).
+None are required, but the runtime MAY support options that start with at least one hyphen.
+Global options MAY take positional arguments (e.g. `--log-level debug`).
+Command names MUST not start with hyphens.
```

Comment:
On Mon, Dec 07, 2015 at 03:34:01PM -0800, Mike Brown wrote:
> > +Command names MUST not start with hyphens.
>
> … You're say, instead, that the spec has certain commands that are
> not hyphenated and they follow the global options.

Right, so what's MUST about it?  There's no way runtime implementors
could get that wrong, since it's a requirement we're making of the
spec itself.  See also my suggested test-case criterion [1].

[1]: https://github.com/wking/oci-command-line-api/commit/527f3c629a7224144c1e5e41a8a197fe0eb5bc00#commitcomment-14835032